sdk/src/OpenTelemetry/Trace.hs

```diff
@@ -125,6 +126,7 @@ module OpenTelemetry.Trace (
   NewEvent (..),
   addEvent,
   inSpan'',
+  getSpanContext,
```

Comment:
We already export `getSpanContext` from `OpenTelemetry.Trace.Core` in the API package. https://hackage.haskell.org/package/hs-opentelemetry-api-0.1.0.0/docs/OpenTelemetry-Trace-Core.html#v:getSpanContext

Is there a reason that we feel like we want it exported from the SDK here?

Reply:
I guess I thought it was odd that we re-export _most_ functions for working with traces from `OpenTelemetry.Trace` here, but not this one. As it is, if you import `OpenTelemetry.Trace`, you get the `NewEvent` constructor but you can't do anything with it.

I guess more generally I'm unsure what our policy is for re-exports from `OpenTelemetry.Trace`. I was thinking along the lines of "everything you need to work with traces", which I think argues in favour of `getSpanContext`?

Reply:
I've dropped this change to keep it uncontroversial